components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/service/DCRManagementService.java

```diff
@@ -343,6 +362,26 @@ private String replaceInvalidChars(String username) {
         return username.replaceAll("@", "_AT_");
     }
 
+    private String createRegexPattern(List<String> redirectURIs) throws DCRException {
+        String regex = "";
```

Comment:
@Pushpalanka Better to write this in StrinBuilder as it is appended inside a loop.
http://stackoverflow.com/questions/14927630/java-string-concat-vs-stringbuilder-optimised-so-what-should-i-do